hypervisor/arch/x86/configs/apl-mrb/ve820.c

```diff
@@ -7,33 +7,33 @@
 #include <e820.h>
 
 const struct e820_entry ve820_entry[E820_MAX_ENTRIES] = {
-	{	/* 0 to mptable */
+	{	/* usable RAM under 1MB */
 		.baseaddr = 0x0UL,
-		.length   = 0xEFFFFUL,
+		.length   = 0xF0000UL,		/* 960KB */
 		.type     = E820_TYPE_RAM
 	},
 
-	{	/* mptable 65536U */
-		.baseaddr = 0xF0000UL,
-		.length   = 0x10000UL,
+	{	/* mptable */
+		.baseaddr = 0xF0000UL,		/* 960KB */
+		.length   = 0x10000UL,		/* 16KB */
 		.type     = E820_TYPE_RESERVED
 	},
 
-	{	/* mptable to lowmem */
-		.baseaddr = 0x100000UL,
-		.length   = 0x1FF00000UL,
+	{	/* lowmem */
+		.baseaddr = 0x200000UL,		/* 2MB */
+		.length   = 0x1FF00000UL,	/* 511MB */
```

Comment:
Then here it should be 512MB, not?

Reply:
Total GPA is 512M, the first 0-1MB is in the first two entries. So here is 511MB.